src/components/image/test.spec.js

```diff
@@ -0,0 +1,67 @@
+
+import React from "react"
+import renderer from "react-test-renderer"
+
+import Image from "./"
+
+const validFluidSrc = {
+  childImageSharp: {
+    fluid: {
+      base64:
+        "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAGABQDASIAAhEBAxEB/8QAFQABAQAAAAAAAAAAAAAAAAAAAAT/xAAVAQEBAAAAAAAAAAAAAAAAAAABAv/aAAwDAQACEAMQAAABomEmDP8A/8QAGBABAAMBAAAAAAAAAAAAAAAAAgEDEhH/2gAIAQEAAQUCbjTs5W69r//EABURAQEAAAAAAAAAAAAAAAAAAAEQ/9oACAEDAQE/ASf/xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAgEBPwFn/8QAGhAAAgIDAAAAAAAAAAAAAAAAAAECMREhUv/aAAgBAQAGPwKL5ZLDd6Lo/8QAHBABAAIBBQAAAAAAAAAAAAAAAQAhQRExYbHB/9oACAEBAAE/Iate+KigVZiOuI2419n/2gAMAwEAAgADAAAAEHv/AP/EABQRAQAAAAAAAAAAAAAAAAAAABD/2gAIAQMBAT8QH//EABYRAQEBAAAAAAAAAAAAAAAAAAABQf/aAAgBAgEBPxDCP//EABoQAQACAwEAAAAAAAAAAAAAAAEAESExgbH/2gAIAQEAAT8QeAVBsG1Xe4UtiQ2Mj0lCyg8PSn//2Q==",
+      aspectRatio: 3.5,
+      src:
+        "/static/2eb58d5c70b74dbe16389cd05d535312/f2e3f/,thisisengineering-raeng-b-ilt-cmx-iw-re-unsplash.jpg",
+      srcSet:
+        "/static/2eb58d5c70b74dbe16389cd05d535312/e2788/thisisengineering-raeng-b-ilt-cmx-iw-re-unsplash.jpg 360w,↵/static/2eb58d5c70b74dbe16389cd05d535312/7d509/thisisengineering-raeng-b-ilt-cmx-iw-re-unsplash.jpg 720w,↵/static/2eb58d5c70b74dbe16389cd05d535312/f2e3f/thisisengineering-raeng-b-ilt-cmx-iw-re-unsplash.jpg 1400w",
+      srcWebp:
+        "/static/2eb58d5c70b74dbe16389cd05d535312/9592d/thisisengineering-raeng-b-ilt-cmx-iw-re-unsplash.webp",
+      srcSetWebp:
+        "/static/2eb58d5c70b74dbe16389cd05d535312/b6bb8/thisisengineering-raeng-b-ilt-cmx-iw-re-unsplash.webp 360w,↵/static/2eb58d5c70b74dbe16389cd05d535312/2aa8d/thisisengineering-raeng-b-ilt-cmx-iw-re-unsplash.webp 720w,↵/static/2eb58d5c70b74dbe16389cd05d535312/9592d/thisisengineering-raeng-b-ilt-cmx-iw-re-unsplash.webp 1400w",
+      sizes: "(max-width: 1400px) 100vw, 1400px",
+      originalName: "thisisengineering-raeng-b-ilt-cmx-iw-re-unsplash.jpg"
+    }
+  }
+}
+
+const validFixedSrc = {
```

Comment:
And in this case could duplicate the import and change `fluid` to `fixed` instead of rewriting all this code.